geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/BinaryExportCommand.scala

```diff
@@ -0,0 +1,63 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo.commands
+
+import scala.collection.JavaConversions._
+import com.beust.jcommander.{JCommander, Parameter, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import org.locationtech.geomesa.tools.accumulo._
+import org.locationtech.geomesa.tools.accumulo.commands.BinaryExportCommand.BinaryExportParameters
+import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
+
+class BinaryExportCommand(parent: JCommander) extends CommandWithCatalog(parent)
+  with ExportCommandTools[BinaryExportParameters]
+  with LazyLogging {
+
+  override val command = "export-bin"
+  override val params = new BinaryExportParameters
+
+  override def execute() = {
+    val start = System.currentTimeMillis()
+    val sft = ds.getSchema(params.featureName)
+    sft.getDtgField.foreach(BinFileExport.DEFAULT_TIME = _)
+    val optAtt = Seq(BinFileExport.getAttributeList(params))
+    val features = getFeatureCollection(Option(seqAsJavaList(optAtt)), ds, params)
+    val exporter: FeatureExporter = BinFileExport(createOutputStream(false, params), params)
+    try {
+      exporter.write(features)
+      logger.info(s"Feature export complete to ${Option(params.file).map(_.getPath).getOrElse("standard out")}")
+    } finally {
+      exporter.flush()
+      exporter.close()
```

Comment:
no need to flush since ```exporter.close()``` should flush. also make it IOUtils.closeQuitely(exporter) so that if an exception is thrown the ds.dispose() still happens. there is probably something better than closeQuitely that will actually log the exception but the key is that control doesn't exit this block before the dispose.